airflow/providers/google/cloud/hooks/datapipeline.py

```diff
@@ -56,14 +41,14 @@ def __init__(
         **kwargs,
     ) -> None:
         super().__init__(
-            gcp_conn_id=gcp_conn_id,
-            impersonation_chain=impersonation_chain,
+            gcp_conn_id = gcp_conn_id,
+            impersonation_chain = impersonation_chain,
```

Comment:
I believe it's proper format to not have the spaces beside the '=' in keyword arguments
https://peps.python.org/pep-0008/#:~:text=PosInt%3A%20...-,Don%E2%80%99t,-use%20spaces%20around

Reply:
i fixed this for both of the hook and operators in each file